features/api-token.feature

```diff
@@ -35,9 +35,8 @@ Feature: User API Token
         Then their token will have that new label
         And the token's 'last used' property will not be updated
 
-    @ignore
     Scenario: Revoke API Token
-        And owns an existing API token
-        When "calvin" revokes the token
+        Given "calvin" owns an existing API token named "tiger"
+        When they revoke the token
```

Comment:
I know originally @r3rastogi commented that it was strange that we put `"calvin"` on all the lines when we weren't actually regex matching/using that value anywhere.

I talked to @petey and he said feature files are meant to be human-readable, and since he thinks `they` is easier to read than having `"calvin"`, I think we've decided to go ahead with `they`.

On a side note, maybe we could use `he` since that is more specific and makes it more obvious only `"calvin"` can revoke the token. This might just be bike shedding, so I'll leave it up to @ian-fox to use his best judgement here.